server/src/main/scala/org/http4s/server/staticcontent/WebjarService.scala

```diff
@@ -0,0 +1,142 @@
+package org.http4s
+package server
+package staticcontent
+
+import java.util.concurrent.ExecutorService
+
+import scalaz.concurrent.{Strategy, Task}
+
+/**
+  * Constructs new services to serve assets from Webjars
+  */
+object WebjarService {
+
+  /** [[org.http4s.server.staticcontent.WebjarService]] configuration
+    *
+    * @param filter To filter which assets from the webjars should be served
+    * @param pathPrefix prefix of the Uri that content will be served from
+    * @param executor `ExecutorService` to use when collecting content
+    * @param cacheStrategy strategy to use for caching purposes. Default to no caching.
+    */
+  final case class Config(filter: WebjarAssetFilter = _ => true,
+                          pathPrefix: String = "",
```

Comment:
Yeah, I basically tried to go along with the other 2 services (which both have it) in staticcontent - but I think it could be scratched.